pyshtools/shclasses/shcoeffs.py

```diff
@@ -1914,7 +1916,7 @@ def rotate(self, alpha, beta, gamma, degrees=True, convention='y',
         if degrees:
             angles = _np.radians(angles)
 
-        if self.lmax > 1200:
+        if self.lmax > 1200 and backend_name() != "ducc":
```

Comment:
This should probably be `=='shtools'`

Reply:
Also fine. I can't predict which recurrence relations future backends will use :-)